.flake8

```diff
@@ -0,0 +1,31 @@
+[flake8]
+exclude =
```

Comment:
After seeing this list for the third time, does `flake8` actually try to parse these non-.py files?

Reply:
It can, depending on how you run it.  This avoids the possibility of it trying.